azext_iot/tests/iothub/certificate/test_iothub_certificate_root_authority_int.py

```diff
@@ -9,23 +9,41 @@
 
 class TestIotHubCertificateRoot(IoTLiveScenarioTest):
     def __init__(self, test_case):
+        self.original_enableRootCertificateV2 = False
```

Comment:
so if the test is canceled/failed before the first show, the teardown will try to change the hub to v1 no matter if it was v1 or v2.

something I use is hasattr to check if an attribute exists for an object. This could solve the issue.

Reply:
oh thanks! will try it out!